### PR TITLE
[TypeDeclaration] Allow multiple declare on DeclareStrictTypesRector

### PR DESCRIPTION
**Pull request by samsonasik**
Ref https://github.com/rectorphp/rector-src/pull/3619/files#r1168951381
See https://3v4l.org/1Kv4V

multiple declare is allowed, with declare(strict_types=1) only allowed on very first stmt

### CODE REVIEW

**Comment by samsonasik** [comment]
not sure why it cause multiple append:

```
+declare(strict_types=1);
+
+declare(strict_types=1);
+
+declare(strict_types=1);
+
+declare(strict_types=1);
+
+declare(strict_types=1);
+
+declare(strict_types=1);
+
+declare(strict_types=1);
+
+declare(strict_types=1);
```

closing for now.

**Comment by samsonasik** [comment]
I see, it seems current node can be checked with:

```php
$currentNode = current($this->file->getNewStmts());
```

File: rules/TypeDeclaration/Rector/StmtsAwareInterface/DeclareStrictTypesRector.php [hunk 1]
```diff
@@ -50,15 +51,24 @@ public function beforeTraverse(array $nodes): ?array
...
+                    return null;
+                }
+            }
         }
 
         $declareDeclare = new DeclareDeclare(new Identifier('strict_types'), new LNumber(1));
         $strictTypesDeclare = new Declare_([$declareDeclare]);
 
+        $rectorWithLineChange = new RectorWithLineChange(static::class, $currentNode->getLine());
+        $this->file->addRectorClassWithLine($rectorWithLineChange);
```
Comment (samsonasik) [inline rules/TypeDeclaration/Rector/StmtsAwareInterface/DeclareStrictTypesRector.php]: @TomasVotruba this is to show applied rule after change:

```diff
 <?php

+declare(strict_types=1);

 use Composer\Semver\VersionParser;
 use Doctrine\Inflector\Inflector;
    ----------- end diff -----------

Applied rules:
 * DeclareStrictTypesRector
```

**Comment by samsonasik** [comment]
All checks have passed :tada: @TomasVotruba it is ready for review.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**